.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
some things in the .vscode folder are not needed, some things would be good to have in the repository for others that use vscode, like the launch profile, think I didn't clean that on my end though and didn't notice it going in, can redo the pr without that and add the .vscode file to .ignore

Reply:
As someone who doesn't use VSCode often for this stuff, I'll defer to what you all need. If it should be in, it's good.

Reply:
.vscode should be added to the .ignore in my opinion.

Reply:
ok, sounds good, in that case we should cancel this pr and do a new one with the changes to .ignore and the tests removed.  Might be easier than correcting the pr.  Thoughts?

Reply:
The only reason I raised the tests (I always feel there is value in asking if there should be tests, and then are the tests adding value), is because they are in memory DB, and that's not necessarily representative.  I can definitely see that we want to make sure we're trying to add the correct data, and that the results are repeatable.  I say we keep the tests, amend the ignore and remove the offending files from the repo (doesn't harm us to have tracking) that would tidy up the PR, correct?